vng_api_common/inspectors/fields.py

```diff
@@ -98,6 +98,31 @@ def field_to_swagger_object(
         return NotHandled
 
 
+class HyperlinkedRelatedFieldInspector(FieldInspector):
+    def field_to_swagger_object(
+        self, field, swagger_object_type, use_references, **kwargs
+    ):
+        SwaggerType, ChildSwaggerType = self._get_partial_types(
+            field, swagger_object_type, use_references, **kwargs
+        )
+
+        if (
+            isinstance(field, LengthHyperlinkedRelatedField)
+            and swagger_object_type == openapi.Schema
+        ):
+            max_length = getattr(field, "max_length", None)
+            min_length = getattr(field, "min_length", None)
+            return SwaggerType(
```

Comment:
aren't we misisng the nullable support here? Or does the `FieldInspector` already do that?

Reply:
It does, I tested it with `allow_null` in the extra serializer kwargs, and it changes the api spec accordingly